src/ui/Footer/Contact.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Icon } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+const Contact = (props) => {
+  if (props.children) {
+    return <div>{props.children}</div>;
+  }
+  return (
+    <>
+      <p className="header">{props.contacts.header}</p>
```

Comment:
@tiberiuichim  I'm not a fan of using Boilerplate code here. The component can be customized using `children` prop.